### PR TITLE
blog: announce the removal of the translated docs

### PR DESCRIPTION
**Pull request by erisu**
### Motivation and Context

* blog post containing announcement of removal.

### CODE REVIEW

File: www/_posts/2021-08-21-removal-of-translated-docs.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Notice of the Removal of the Translated Docs"
+categories: announcements
+tags: news
+---
+
+We like to announce that we have deprecated and removed all translated docs.
```
Comment (dblotsky) [inline www/_posts/2021-08-21-removal-of-translated-docs.md:10]: "We like" -> "We'd like"

Comment (dblotsky) [reply www/_posts/2021-08-21-removal-of-translated-docs.md:10]: But also, please consider giving this some time; first, deprecate, **then** remove.

File: www/_posts/2021-08-21-removal-of-translated-docs.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+We had been providing translated docs since Cordova CLI 3.1.0 but, the number of resources has decreased to a point where it affects our ability to maintain them. The maintenance of these translated docs started to diverge around Cordova CLI 6.x.
```
Comment (dblotsky) [inline www/_posts/2021-08-21-removal-of-translated-docs.md:12]: When you say "the number of resources", do you mean "the number of resources required to maintain them"?

File: www/_posts/2021-08-21-removal-of-translated-docs.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+We have updated our webserver to redirect the language docs to their English counterparts. A 404 page will display only when the English counterparts do not exist. This issue occurs when translated docs continue to show deprecated content. For example, the BlackBerry 10 platform was deprecated and removed from Cordova CLI 8.x but, the translations were not updated and still show the deprecated platform.
+
+We like to thank the people who took the initiative in creating the translated docs and those who had put in their time and effort into maintaining them.
```
Comment (dblotsky) [inline www/_posts/2021-08-21-removal-of-translated-docs.md:16]: "We like" -> "We'd like"

File: www/_posts/2021-08-21-removal-of-translated-docs.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+For non-English speakers, we received feedback from various users that using services such as Google Translate to translate the English documentation into their language has produced better readable content over the existing translated docs. If you are using Google Chrome or Microsoft Edge, you could right-click and translate the page into your language.
```
Comment (dblotsky) [inline www/_posts/2021-08-21-removal-of-translated-docs.md:18]: Nitpick: you can also do this in Safari. :)

File: www/_posts/2021-08-21-removal-of-translated-docs.md (hunk 1 above)
Comment (erisu) [reply www/_posts/2021-08-21-removal-of-translated-docs.md]: I changed it to `We'd` as suggested but, I am making a note that we are removing. We are not talking about `would like to remove`. So the tone of this is wrong and lying.

File: www/_posts/2021-08-21-removal-of-translated-docs.md (hunk 3 above)
Comment (erisu) [reply www/_posts/2021-08-21-removal-of-translated-docs.md]: I changed it to We'd as suggested but, I am making a note that we are removing. We are not talking about would like to remove. So the tone of this is wrong and lying.

File: www/_posts/2021-08-21-removal-of-translated-docs.md (hunk 2 above)
Comment (erisu) [reply www/_posts/2021-08-21-removal-of-translated-docs.md:12]: `the number of resources` has decreased to a point where it affects our ability to `maintain them`.

Resources = people

File: www/_posts/2021-08-21-removal-of-translated-docs.md (hunk 4 above)
Comment (erisu) [reply www/_posts/2021-08-21-removal-of-translated-docs.md:18]: > If you are using Google Chrome or Microsoft Edge, you could right-click and translate the page into your language.

Safari did not fit well with this sentence about right-clicking. Safari does have some translation features but is limited to installed preferred languages, and not all are displayed. I test with English, Chinese & Japanese installed locally but, I could not translate the site into Japanese.

File: www/_posts/2021-08-21-removal-of-translated-docs.md (hunk 3 above)
Comment (erisu) [reply www/_posts/2021-08-21-removal-of-translated-docs.md]: I changed it to`We'd` as suggested.

File: www/_posts/2021-08-21-removal-of-translated-docs.md (hunk 1 above)
Comment (erisu) [reply www/_posts/2021-08-21-removal-of-translated-docs.md]: I changed it to`We'd` as suggested.

**Comment by erisu** [comment]
@purplecabbage I finished with this PR too. Merge it in when ready.

File: www/_posts/2021-08-21-removal-of-translated-docs.md (hunk 4 above)
Comment (dblotsky) [reply www/_posts/2021-08-21-removal-of-translated-docs.md:18]: You can right-click on the "translate" icon at the top. :) But yeah, fair, if Safari is an outlier in the number of languages, then it won't quite fit in.

File: www/_posts/2021-08-21-removal-of-translated-docs.md (hunk 3 above)
Comment (dblotsky) [reply www/_posts/2021-08-21-removal-of-translated-docs.md]: Thanks!

File: www/_posts/2021-08-21-removal-of-translated-docs.md (hunk 1 above)
Comment (dblotsky) [reply www/_posts/2021-08-21-removal-of-translated-docs.md]: Thanks!

**Review by NiklasMerz — APPROVED** [review]

**Comment by erisu** [comment]
This PR is closed by #1199

**Pull request closed without merging**